std/conv.d

```diff
@@ -5519,7 +5519,7 @@ pure nothrow @safe @nogc unittest
 }
 
 // Test attribute propagation for UDTs
-pure nothrow @safe /* @nogc */ unittest
+pure nothrow @safe /+@nogc+/ unittest
```

Comment:
Unrelated change - remove?

Reply:
Made it easier to exclude ranges of unittests, while trying to track down a particular issue.
But reverting the change.